### PR TITLE
Pin build requirements

### PR DESCRIPTION
**Pull request by hneiva**
## Description

These requirements are used during builds.
Pinning minimizes some security risks and unwanted bugs introduced by upstream changes.

## Reference

N/A

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: taskcluster/scripts/requirements.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# These requirements are used in build scripts
+
+multidict>=5.0.0 
+yarl>=1.6.0
+taskcluster==53.1.0
```
Comment (hneiva) [inline taskcluster/scripts/requirements.in:5]: Note I'm upgrading `taskcluster` from `52.0.0` to `53.1.0`
I'm not sure if pinning and upgrading packages belong in the same PR. Let me know if we should split them.

**Comment by hneiva** [comment]
Closes #7283

**Comment by ahal** [comment]
Lol, so we now have:

- requirements.in
- requirements.txt
- taskcluster/requirements.in
- taskcluster/requirements.txt
- taskcluster/scripts/requirements.in
- taskcluster/scripts/requirements.txt

Maybe we should move these into a single `requirements` directory? E.g:

- requirements/decision.in
- requirements/decision.txt
- requirements/build.in
- requirements/build.txt
- etc...

This seems like an improvement though, so possibly it could be a follow-up.. I dunno.

**Review by ahal — APPROVED** [review]
I'll stamp this from a Releng perspective, though I think we should block on review from Lesley or someone from the VPN team.

**Comment by hneiva** [comment]
@ahal what's trying to rerun jobs? Is there a way to find out?

**Review by lesleyjanenorton — APPROVED** [review]
R+ pending conflict resolution. thanks!

**Comment by strseb** [comment]
I think this has bitrotted a bit away, i'm closing this. Please re-open if thats not the case

**Pull request closed without merging**